syncer/src/index.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * Contains request routing logic.
+ */
+
+import {App} from '@octokit/app';
+import {Octokit} from '@octokit/rest';
+
+import {handleRequest} from './handler';
+import {pullRequestClosedHandler} from './webhook';
+
+// Secrets defined by `wrangler secret`.
```

Comment:
Why are we using wrangler to manage secrets instead of a gh actions environment? To be honest I don't know what wrangler is.

Reply:
This raises an important enough question to elevate this to the main discussion, see below...